pyaedt/modules/MaterialLib.py

```diff
@@ -463,15 +463,23 @@ def add_material_sweep(self, swargs, matname):
         return index
 
     @pyaedt_function_handler()
-    def duplicate_material(self, material, new_name):
+    def duplicate_material(self, material_name, new_name=None, props=None):
         """Duplicate a material.
 
         Parameters
         ----------
-        material : str
+        material_name : str
             Name of the material.
         new_name : str
             Name for the copy of the material.
+        props : list
+            List of properties that will be parameterized when the material is duplicated.
+            Parameterized properties will have project scope. Allowed items are:
+            - 'permittivity'
+            - 'permeability'
+            - 'conductivity'
+            - 'dielectric_loss_tan'
+            - 'magnetic_loss_tan'
```

Comment:
```suggestion
            List of properties to parameterize when duplicating the material.
            Parameterized properties have project scope. Options are:

             - `'permittivity'`
            - `'permeability'`
            - '`conductivity'`
            - '`dielectric_loss_tan'`
            - '`magnetic_loss_tan'`
```
Sometimes parametrize is used and sometimes parameterize is used. We should pick one and be consistent within the library.

Reply:
updated